docs/mesos/RpmInstallMesos.md

```diff
@@ -0,0 +1,51 @@
+# Create and Install the Mesos + Net-Modules RPM
+This guide will walk you through building and installing Mesos + Net-Modules from RPMs. These RPMs will automate the installation of everything needed to run Mesos + Net-Modules in your cluster.
+
+## Build and Install the RPM
+The Mesos + Net-Modules RPMs can easily be built and installed via the [net-modules repository](https://github.com/mesosphere/net-modules).
+> Note, Extra Packages for Enterprise Linux (EPEL) must be installed before installing Mesos + Net-Modules. You can download this package by calling `sudo yum install epel-release`
+
+    $ git clone https://github.com/mesosphere/net-modules.git
+    $ cd net-modules
+    $ git checkout integration/0.25
+    $ make builder-rpm
+    $ sudo yum install packages/rpms/RPMS/x86_64/*.rpm
+
+## Configure and Run Master
+
+We will be need to set the correct environment variables for the master. These environment variables are interpreted as command line arguments in the mesos-master application at runtime.
```

Comment:
"...are interpreted the same ways as the command line argument with corresponding name..."